### ISSUE TITLE
SA1121 not reported for type arguments in using alias declarations

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#969]
In the following line, `System.Int32` can be replaced by `int`. SA1121 should be reported, but it currently is not.

```csharp
using IntAction = System.Action<System.Int32>;
```

**Comment by pdelvo** [comment]
grabbed

**Comment by sharwell** [comment]
If you're allowed to create a generic `static class`, can you also write a test for this with a `using static` directive?
